eng/pipelines/client.yml

```diff
@@ -13,5 +13,5 @@ pr: none
 jobs:
 - template: templates/jobs/archetype-sdk-client.yml
   parameters:
-    ProjectFile: $(ProjectFile)
+    ServiceDirectory: $(ServiceDirectory)
```

Comment:
In order to keep our existing pipelines working we will need to add the ServiceDirectory variable and pass `*` as the default value for now.

Reply:
Another option would be to conditionalize the template so that if ServiceDirectory isn't supplied it builds the universe.